testing_and_setup/compass/ocean/scripts/vertical_grid/define_vertical_grid_64_layers.py

```diff
@@ -1,12 +1,14 @@
 #!/usr/bin/env python
 
-import subprocess
+import make_vertical_grid
 
 # Standard 64 layer vertical grid
-subprocess.check_call(['./make_vertical_grid.py', 
-    '--output_file_name', 'vertical_grid.nc',
-    '--num_vert_levels', '64',
-    '--bottom_depth', '5500',
-    '--layer1_thickness', '2',
-    '--maxLayer_thickness', '200',
-    '--plot_vertical_grid'])
+make_vertical_grid.create_vertical_grid(
+     5500,            # bottom_depth
+     64,              # num_vert_levels
+     2,               # layer1_thickness
+     200,             # maxLayer_thickness
+     True,            # plot_vertical_grid
+     maxit=1000,
+     epsilon=1.0e-2,
+     outFile= 'vertical_grid.nc')# output_file_name  
```

Comment:
In general, comments on lines are discouraged in python.  this one is not needed.